.travis.yml

```diff
@@ -11,7 +11,7 @@ before_script:
   - npm run bootstrap
 
 script:
-  - npm run test:ci
+  - npm run test
```

Comment:
`script:` and `-npm run test` can be removed in it's entirety as that's the default for travis. It was only added because we were running `npm run test:ci`.

Reply:
Good catch. Even though `npm run test` is the same as `npm test`, I prefer to use `npm test`.